tools/statistical_ensemble_test/single_run.sh

```diff
@@ -293,7 +308,7 @@ case $MACH in
   ;;
 esac
 
-NewCaseFlags="$NewCaseFlags -res $RES -compset $COMPSET"
+NewCaseFlags="$NewCaseFlags -res $RES -compset $COMPSET --run-unsupported --project $ACCOUNT"
```

Comment:
Double-dashes are required in most cases so fix as many as possible and note here any that cannot be fixed. See PR #1393.